<launch>
    <!-- RQT Battery perspective -->
    <node name="bender_dashboard" pkg="rqt_gui" type="rqt_gui" output="screen" args="--perspective-file $(find uchile_rqt_batteries)/config/bender_dashboard.perspective" />

    <node name="chest_batt" pkg="uchile_rqt_batteries" type="notebook_battery_pub.py">
        <param name="serial_number" value="chest" />
    </node>
    
    <node name="left_arm_batt" pkg="uchile_rqt_batteries" type="dxl_battery_pub.py">
        <param name="serial_number" value="l_arm" />
        <remap from="motor_states" to="/bender/motor_states/l_port"/>
    </node>

    <node name="right_arm_batt" pkg="uchile_rqt_batteries" type="dxl_battery_pub.py">
        <param name="serial_number" value="r_arm" />
        <remap from="motor_states" to="/bender/motor_states/r_port"/>
    </node>

    <node name="base_batt" pkg="uchile_rqt_batteries" type="voltage_battery_pub.py">
        <param name="serial_number" value="base" />
        <param name="min_voltage" value="11.5" />
        <param name="max_voltage" value="13" />
        <remap from="battery_voltage" to="bender/nav/base/battery_voltage"/>
    </node>
    

</launch>